OrbitService/LinuxUtils.cpp

```diff
@@ -166,4 +176,24 @@ ErrorMessageOr<std::string> GetExecutablePath(int32_t pid) {
   return std::string(buffer, length);
 }
 
+outcome::result<std::string> FileToString(
```

Comment:
ErrorMessageOr?

Reply:
I change that in the 3rd of these commits. I could move it to commit 1, but I dont think it actually matters?

Reply:
it's fine as it is.